src/index.css

```diff
@@ -23,7 +23,6 @@ body {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
-  height: 430vh;
```

Comment:
FYI: This means "virtual height". 0-100 anyway is 0-100% of the height of the browser window. If you use percentage instead, it's the percent of the "parent" component. So with percent, if you wanted to set height on `grandparent > parent > this one`, both the grandparent and parent would need `height: 100%` if you wanted "this one" to be full height.